core/io/resource_loader.cpp

```diff
@@ -818,6 +818,8 @@ Ref<Resource> ResourceLoader::_load_complete_inner(LoadToken &p_load_token, Erro
 		load_task_ptr = &load_task;
 	}
 
+	thread_load_mutex.unlock();
```

Comment:
Given the scope I'd use the lock, like so:
```suggestion
	p_thread_load_lock.temp_unlock();
```